client/server/pages/index.js

```diff
@@ -681,7 +681,39 @@ module.exports = function() {
 		next();
 	} );
 
+	function handleSectionPath( section, sectionPath, entrypoint ) {
```

Comment:
I'm curious what's the purpose behind moving that one.

Reply:
Well, it's moved here to make the code a bit tidier after reusing the `handleSectionPath` function in Jetpack Cloud flow.

On `master`, this function is used only once, [later in the code](https://github.com/Automattic/wp-calypso/pull/39607/files#diff-13202bbd2bfb14a1570c13babd9e8444R864). Since [we're now using it earlier as well](https://github.com/Automattic/wp-calypso/pull/39607/files#diff-13202bbd2bfb14a1570c13babd9e8444R714), I thought it'd be nice to have the function definition before it is first used (even though it'll be hoisted).

We can move it back, though.